ROADMAP.md

```diff
@@ -1,5 +1,38 @@
 # Kubeflow Roadmap
 
+## Kubeflow 1.8 Release, Planned: May 2023
+The Kubeflow Community plans to deliver its v1.8 release in Oct 2023 per this timeline(pending link). The high level deliverables are tracked in the [v1.8 Release](https://github.com/orgs/kubeflow/projects/58/) Github project board. The v1.8 release process will be managed by the v1.8 [release team](https://github.com/kubeflow/community/blob/a956b3f6f15c49f928e37eaafec40d7f73ee1d5b/releases/release-team.md) using the best practices in the [Release Handbook](https://github.com/kubeflow/community/blob/master/releases/handbook.md)
+
+### Themes
+* Kubernetes 1.26 and 1.27 support
+* Kubeflow Pipelines v2.0 release
+* Kubeflow security (TBD)
+* Kubeflow upgrade process (TBD)
```

Comment:
We have listed upgrades on the roadmap before and have not delivered.   I am cautious to list upgrades without qualifier (stretch goal) or some well defined step towards upgrade i.e. list breaking changes, identify to/from dependencies and statements for upgrading a limited part of KF.   The documentation for an upgrade could be extensive as there are potentially a lot of choices.

Reply:
Agree, after some discussion with the release team, I think I can narrow this down. I'll update this.